modules/compliance-anatomy.adoc

```diff
@@ -109,6 +110,28 @@ NAME                                           SCHEDULE    SUSPEND   ACTIVE   LA
 
 For the most important issues, events are emitted. View them with `oc describe compliancesuites/<name>`. The `Suite` objects also have a `Status` subresource that is updated when any of `Scan` objects that belong to this suite update their `Status` subresource. After all expected scans are created, control is passed to the scan controller.
 
+Sometimes, the `ComplianceSuite` CR cannot create a scan if the scan exceeds the default memory limit of the Compliance Operator. Exceeding this limit can result in an Out of Memory (OOM) error message. You can increase the memory for the Compliance Operator by setting custom resource limits.
```

Comment:
It looks like we still have a similar section at the bottom of this page:

http://file.emea.redhat.com/dfitzmau/01-25-23/BZ1990481-R/security/compliance_operator/compliance-operator-troubleshooting.html#operator-resource-constraints_compliance-anatomy-sections

Do we want to keep that separate from this? Or do we want to take the opportunity to consolidate that section too?